lola.html

```diff
@@ -581,14 +589,57 @@ <h4>Example Copied Object</h4>
   "@context": ["https://www.w3.org/ns/actvitystreams",
                {"@language": "en"}],
   "id": "https://newsite.example.org/amy/items/02751cab-7dd7-416b-905b-...",
-  "previously": ["https://rhiaro.co.uk/2016/05/minimal-activitypub"]
+  "previously": ["https://rhiaro.co.uk/2016/05/minimal-activitypub"],
```

Comment:
is this a neologism, btw, or is the "previously" semantic defined in some FEP or extension already? maybe a mental note to add it to the `@Context` file in final t-crossing and i-dotting stage